src/coreclr/tools/superpmi/superpmi-shim-collector/CMakeLists.txt

```diff
@@ -55,8 +55,14 @@ else()
     set(SPMI_COLLECTOR_EXPORTS_LINKER_OPTION ${EXPORTS_LINKER_OPTION})
 endif (CLR_CMAKE_TARGET_WIN32)
 
+if(CLR_CMAKE_TARGET_BROWSER)
```

Comment:
All of the superpmi should be excluded for wasm, it is a JIT testing framework that intercepts calls from coreclr to JIT. So it doesn't make sense to have it for wasm.